guide/14-deep-learning/how_named_entity_recognition_works.ipynb

```diff
@@ -0,0 +1 @@
AxBvqKceJdjDMHoTwoFrV0O53700UeLpgr5u/HGG6Uk/E9IFK2VdqcaF1988aJppAHrUiObUydQvQbfj6gGyW1oSG4zVzQNH1CNu+666/333y/O2wKUI22deeaZ0pDNZdGaGBJMz5HGGWecBRZYoGiqDTAfTv/Xv/61OK+I+vPPPy+xxBLF+RgAIdm5c+ctt9wSt/j3v/9dtFZU0Zy4BmpEr8uHN6ME5NsbbrihNT/yEfrkXcV546BPgjUn8Y477rjjjTcenvT2228XTYMhe7/xxhsY0hRTTFE0NQ9TTjnljhUgzXjYQQcdJMdeffXVcZVUYrk5tv7111+bs4qPP/54//33536vv/46Yle0DsYEE0ywyiqrXHPNNSjjhhtuuN9++/Xt21fMXnLJJdHho48+sie3c8Pd11tvPXn42GOP1XjLLbdEh9Eb4g4/fvPNN4vzMQNjBClcd911HaA40QKi7v7770cWZ5ttNv5dhpZIe/DBB6eZZpqykukpwX322Wf2vl4dV0esHKT9j7dJvv/e1eoNn9i2y4y70LU60ev0ySefPPXUU6WbphOZ6a644orLL7+8ztTVMIKpySAzmtGx2ctLbtQItozVj0shPiwSV7/44gtZNdpjQK91BjRF9RohOhgkTktJoI5C5BoCxGh1PtpCG1pKdTmOdtLefPPNE088cWPv5pv9q6++MqARKNm9RtBCjKJHBbqRxwJdBWKU+oFyENPFgavRaEByksdd2sOO1S1NGKU+aDsGN1Go3amhyhFMKtFr1FN7jB+XzGsJpYoca4l2Lbo5sK64pRQ+Ojh2Y/QHVw2updoEjUHnUnUxeNxlNKd11Eh4k1pdOal2Sot7XXXgKk+IVTgOnXA8t5fHMSDooIUAxXmTCK3qb/BqZw7EQkKBXquvuuTeUHhc1VNjXLVe7bHw6ANWV9/HjFkqSs9SvRNNNNGqq66KPVx88cXRArQhrq+//nojN+1COkgUDzzwgFtKqRqDDtRYilHnFiIxRyjBVcelkBALLFXkanGhAuM0psBqML0bCaB/GFpn6jU4/3RMbw5Y3wjRqHNxcwXVdgwhiwtViMFd1TMW6IA+b7rpJktoWp8ffvgh9jNw4MDGllBisskmW2eddehQ+q1WIxDbXCrF8LyXYqO70korcYySdljyaaed9vDDD1d7YINAJQ844IBXX3216VXQcMeOHfkP5rf00ksXrYMx7bTTbrvtttNNN12ctm/f3opmmGGG559/PloYi6t36tRpkkkmcYpEYpbzzDMPOaNDfRA77grz1XfvCOrSrHHKavyWVt3FHxjXOGVLZLwGFWKB5tKt+ion1N84BnfqUulUoL+Ro2dABzOGzKUAbgk/N4UDMpQyG197yGZAB9XilSFgEJNW3zhKYIwghZyYr997772spYXxbNlfeeWVZZddVkg/9NBD7BqdP/nkEwG54oorxqmahzvusMMOSy655MILL/y3v/3NDu/RRx8NLzfOY489Nvvss59yyik2Uihm+Vla7mUXqGXBBRc0y4EHHvif//wnLgVIhetcddVVm2666Y033sirGktkespNhx566NZbb23PKnhiFSWcPvHEE4sttpgZeSE5HV966aUu8dTbbrutR48ef/nLXxZaaCHU6u677y6LmdXJFK4usMAC5LTq8847T7SIgccff9ySDauPARdffPHLLrvMjZY533zzxe0BGXOOOeaITw27EaWea665KKRPnz6WD9opSlBdeOGF2DkxYJdddnnxxRdjIV6ltj322EPhdImSe/Xq9cfQFRalasrLjT0mJN5mm21GJPvgI488kv4txCy33norYaKPA8q3wdVOD5bzj3/8w0JoMjpYQteuXQ2i7lrIcsstZ+MujDXOP//8r7322r777mtYx7LwBx98QNq9995bC71ttdVWzfxaiRTDA//0pz+dddZZ9913384770wS67VHp+ewvkRDdSS57rrr7O8lcZ4T96phxxxzjBKyyCKLkPD444+Ph7hSD+vsvvvultOtWzfHCp5MxK8WXXRRHSxku+2244T2907Zwi0mXWONNT7++GMtTUBnEXH00Uevt956FbstxP9jXu3kDDcLcLaTTjrJol544QUCKDarrLIK3dK8xmWWWeaMM87gD8Ln9ttv18LfnnvuuTvuuINijWON1GJ19BO20yL66Ll8btEEzH7nnXealNIMvvbaa1d/5cJCpGZGJ5IB+YlZBEtc5Wbu5XV83hqJaiGlclhcAqExHrvXXnu5Vx++YRVlaeHD+p999tnmNT7N85ABAwaUoYoXGta+tAzzscf+4w8HHHTQQWLB/pCZys51QHgJiuZ5+FtvvVVNxOuDG1x77bXrr78+V6fMI444osw89GkJ3IPmCWnfazkCp7z67LPPWiYfo0AOQ7a4BLFA3GLllVd2Lw+MXFFcrgIBdCMAZ77hhhsMtfrqqwsZYoi7pZZaisW7d+++2mqr2e9FY5lvrZTDMHrPnj3DFvzZquNqCd04WL9+/QSg2IyKi9BwGxG62267Pf3008ZpTJ9MwPFE7gUXXCAWynxYH2w000wzyUtkVv6L1oo2RBNrijK8qmhtOSwEMK1xxhmnaKqwvZ122um44457++23myB87GWZisJFF13UxCrIf8IJJ2y88cbB6oYKOqTJzp07x+n4448/88wzy968zqqFzDsVTDXVVNGhDiyHA4gmAcI6jKh6iutqW8gPQr78aEGc7rnnnv/6178kc97FA4W8gFUZN9lkE568xBJLWIWcE7dU49133+VOshklFE2VrHjyySfz1ZdeeilEYiwRRB6xSRtipNQt2RjXjmLzzTcns6hRcWRgwRKPUVF2YoiI+D4cFbER/99ggw3IptqqLCeeeGL52VCRIm+jFvYSvNFyRuoXTEc4Rn9SCGJbVXv55ZfDq1hOGR533HGlNn7DJ+KjxHEgU0tYTuUOtQqR4jFSvATHwBI918EF/xi3AnfhEPIjSiH3aRE/vXv3V
```

Comment:
<p>It would be great if you can talk a bit about the context and goal of this analysis. Looks like you are extracting geo addresses and weapons from plain text and in order to detect those, you need to hand label it. Correct?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/556/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Correct. The context of the analysis is provided in the <a href="https://github.com/Esri/arcgis-python-api/blob/5c121194949d774cd08ed22963c4434f5ba52ba1/samples/04_gis_analysts_data_scientists/information-extraction-from-madison-city%E2%80%99s-crime-incident-reports-using-deep-learning.ipynb" target="_blank">sample notebook </a></p><p> </p>